src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -123,6 +128,7 @@ public async Task<IActionResult> GetSeriesMetadataAsync([FromHeader(Name = IfNon
             return CreateResult(response);
         }
 
+        [Produces(KnownContentTypes.ApplicationDicom, new[] { KnownContentTypes.MultipartRelated + TypeSeperator + KnownContentTypes.ApplicationDicom })]
```

Comment:
Can this produce multipart responses?

Reply:
https://github.com/microsoft/dicom-server/blob/main/docs/resources/conformance-statement.md#retrieve-an-instance

From here it looks we do produce a multipart/related